.circleci/config.yml

```diff
@@ -0,0 +1,49 @@
+version: 2
+
+jobs:
+  test:
+    machine:
+      docker_layer_caching: true
+    working_directory: ~/codeclimate/codeclimate-rubocop
+    steps:
+      - checkout
+      - run: make image
+      - run: make test
+
+  release_images:
+    machine:
+      docker_layer_caching: true
+    working_directory: ~/codeclimate/codeclimate-rubocop
+    steps:
+      - checkout
+      - run:
+          name: Validate owner
```

Comment:
circle 1 has this nice feature https://github.com/codeclimate/codeclimate-rubocop/blob/master/circle.yml#L24. 

Unfortunately not available in circle2 so we're the validation within the job itself. 

`circleci step halt` is a nifty trick to exit the job without failing the build. I learn it from this [post](https://discuss.circleci.com/t/useful-tips-and-best-practices-when-migrating-to-circleci-2-0/23075).

Reply:
That is neat!